src/main/java/net/openhft/chronicle/queue/impl/single/TableDirectoryListing.java

```diff
@@ -126,8 +126,10 @@ public void refresh(final boolean force) {
             if (!INITIAL_MAX_FILENAME.equals(maxFilename))
                 max = fileNameToCycleFunction.applyAsInt(maxFilename);
 
-            if (currentMin0 == min && currentMax0 == max)
+            if (currentMin0 == min && currentMax0 == max) {
+                modCount.addAtomicValue(1);
                 return;
+            }
```

Comment:
If someone did a force refresh, even if the min and max cycles didn't change, we should bump the modCount to trigger a rebuild of the cycle tree. This allows us to move past missing files.